addons/event/report/event_event_templates.xml

```diff
@@ -256,7 +276,28 @@
     <div class="o_event_badge_ticket_wrapper" t-att-style="'background-image: url(%s);' % badge_image_url if badge_image_url else ''">
         <div class="position-relative h-100">
             <h3 class="fw-bold text-center" t-field="event.name"/>
-            <div class="text-center mt-5 py-2">
+            <div class="text-center o_event_badge_font_small">
+                <span itemprop="startDate" t-field="event.date_begin"
```

Comment:
Hum I suppose we could use the method to get the event date with the timezone here 🤔